### PR TITLE
[DEV-7400] Copy updates - Award spending by recipient

### PR DESCRIPTION
**Pull request by afnazu**
**High level description:**

Implement copy updates for award spending by recipient section

**JIRA Ticket:**
[DEV-7400](https://federal-spending-transparency.atlassian.net/browse/DEV-7400)

**Mockup:**
[copy document](https://pmeonpremise.bah.com/sites/data-transparency/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2Fdata%2Dtransparency%2FShared%20Documents%2F01%20Data%20Transparency%2F09%20USAspending%2FCares%20Act%2FCopy)

[Component Library PR](https://github.com/fedspendingtransparency/data-transparency-ui/pull/101) to fix 'same key' console errors

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [n/a] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [n/a] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [n/a] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [n/a] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [n/a] Design review complete `if applicable`
- [n/a] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

**Comment by brett-varney** [comment]
@afnazu This branch is throwing a bunch of warnings on the page:

Warning: Encountered two children with the same key, `[object Object]`. Keys should be unique....

I don't see what's causing it offhand.

File: src/_scss/pages/covid19/recipient/recipientContainer.scss [hunk 1]
```diff
@@ -4,6 +4,11 @@
   .count-tabs {
     &.recipient__tabs-container {
       width: 100%;
+      .usa-dt-tab-list {
+        z-index: 15;
+        overflow-x: visible;
```
Comment (ebdabbs) [inline src/_scss/pages/covid19/recipient/recipientContainer.scss:9]: Screen Shot 2021-06-30 at 12 43 49 PM
This change is causing the tabs to overflow in mobile - was there another issue you were trying to fix?

Comment (ebdabbs) [reply src/_scss/pages/covid19/recipient/recipientContainer.scss:9]: Ohhh I I see it is for the tooltip in the tab. I brought this up as an anti-pattern and accessibility issue on another ticket. Might be a larger discussion. I'll see if I can find that Slack thread!

**Review by ebdabbs — CHANGES_REQUESTED** [review]
I'm going to ask if we can remove the tooltip from the tab and put the note somewhere else

**Review by ebdabbs — APPROVED** [review]

**Pull request merged**